cds/modules/deposit/static/templates/cds_deposit/types/video/form.html

```diff
@@ -137,6 +137,25 @@ <h4 class="mb-5 mt-0">
                 <p class="mb-0">Type</p>
                 <p ng-if="$ctrl.cdsDepositCtrl.cdsDepositsCtrl.master.metadata.type"><span class="label label-warning">{{$ctrl.cdsDepositCtrl.cdsDepositsCtrl.master.metadata.type}}</span></p>
               </div>
+              <div class="col-md-12">
```

Comment:
this is under the small video player right?

Reply:
Yes, I've added a couple screenshots to the issue.